src/core/Akka/Serialization/Serialization.cs

```diff
@@ -145,14 +137,26 @@ private Serializer GetSerializerByName(string name)
         public ActorSystem System { get; }
 
         /// <summary>
-        /// TBD
+        /// Adds the serializer to the internal state of the serialization subsystem
         /// </summary>
-        /// <param name="serializer">TBD</param>
-        /// <returns>TBD</returns>
+        /// <param name="serializer">Serializer instance</param>
+        [Obsolete("No longer supported. Use the AddSerializer(name, serializer) overload instead.", true)]
```

Comment:
Left out a "deadline" version here. Or would you rather have i put in something like. will be removed in 1.4 or something ?

Reply:
This is fine as-is.